File: src/main/java/things/WeatherUtil.java
package things;

public final class WeatherUtil {
    public static void update(int degrees){
        String description;
        TemperatureColor color;

        if (degrees > 30){
            description = "hot";
            color = TemperatureColor.RED;
        } else if (degrees > 10) {
            description = "warm";
            color = TemperatureColor.ORANGE;
        } else {
            description = "cold";
            color = TemperatureColor.BLUE;
        }
    }
}

File: src/main/kotlin/weirdthings/WeatherUtil.kt
package weirdthings

object WeatherUtil {
    fun update(degrees: Int): Pair<String, TemperatureColor> {
        return when {
            degrees > 25 -> Pair("hot", TemperatureColor.RED)
            degrees > 10 -> Pair("warm", TemperatureColor.ORANGE)
            else -> Pair("cold", TemperatureColor.BLUE)
        }
    }
}